network.tf

```diff
@@ -1,10 +1,10 @@
 resource "openstack_networking_network_v2" "slurm_network" {
-    name = "${var.image_name_prefix}-net"
```

Comment:
All of this stuff is quite new to me. So let me know if I am just being annoying. But if the suffix is blank this would add a '-' at the end. Or am I missing something.

Reply:
Yeah, it would. It's set by default though… And I don't think it's a train smash if it ends in a "-" :-).